### ISSUE TITLE
Disable arriving/landing aircraft if taxi route is too small for that aircraft size

### ISSUE THREAD

**Issue by togfoxy** [felix-b/atc#208]
Assumption: Planes that land and can't find a taxi network that is of the planes size or larger will taxi off the end of the rway and go for a picnic.

Before spawning an airplane for arrival/approach, check that that plane is physcially able to use that runway/airport and abort spawn if not.

**Comment by togfoxy** [comment]
(should also check that there is an empty gate for them I guess)

**Comment by felix-b** [comment]
I'd say the plugin should test an airport for setup requirements (taxi net, gates, etc), then:
(1) as you suggest - degrade gracefully
(2) warn the user about limited capabilities on that specific airport

**Comment by felix-b** [comment]
I think we're going to meed all kinds of problems in airport setups, so it looks like this is an ongoing task. But - in this ticket, we can create the "airport compatibility test" infrastructure with some basic checks - adding more checks as we go.

**Comment by togfoxy** [comment]
Yep. Suggest tiny steps. We'll add more rules and more robustness with new issues and priorities accordingly.
